added calls onEnterForeground & onExitForeground

n3wc opened the pull request:
per comscore support they recommended using onEnterForeground &
onExitForeground in our apps instead of onUxActive & onUxInactive

n3wc committed:
added calls onEnterForeground & onExitForeground

per comscore support they recommended using onEnterForeground &
onExitForeground in our apps instead of onUxActive & onUxInactive

stephenfeather commented:
Could you add the documentation/examples demonstrating this in the app lifecycle events in your request so they all go in together as a single PR.

n3wc committed:
Update index.md

added entries for onEnterForeground, onExitForeground & additional information about the indended usage of onUxActive

n3wc commented:
sorry for missing that, its been added
